Tests/KituraTests/KituraTestBuilder.swift

```diff
@@ -84,6 +86,16 @@ class ServerTestBuilder: RequestTestBuilder, AssertionTestBuilder {
                 })
             }
         }
+        
+        init(_ test: KituraTest, _ method: String, _ path: String, _ URLEncodedObject: String) {
```

Comment:
URLEncodedObject - > urlEncodedString I think, to be consistent with the `request()` function.